Makefile

```
@@ -223,12 +223,12 @@ else
 endif
 
 
-examples/vk10/vk10: examples/vk10/vk10.c examples/vk10/vkinit.h RGFW.h
+examples/vulkan/vulkan: examples/vulkan/vulkan.c RGFW.h
 ifneq ($(NO_VULKAN), 1)
-	glslangValidator -V examples/vk10/shaders/vert.vert -o examples/vk10/shaders/vert.h --vn vert_code
-	glslangValidator -V examples/vk10/shaders/frag.frag -o examples/vk10/shaders/frag.h --vn frag_code
+	glslangValidator -V examples/vulkan/vert.vert -o examples/vulkan/vert.h --vn vert_code
+	glslangValidator -V examples/vulkan/frag.frag -o examples/vulkan/frag.h --vn frag_code
 
-	$(CC)  $(CFLAGS) -I. $< $(VULKAN_LIBS) -o $@
+	$(CC)  $(CFLAGS) -I. -Iexamples/vulkan $< -lm $(VULKAN_LIBS) -o $@
```

come to think of it this should probably be here
```suggestion
	$(CC) $(CFLAGS) $(CUSTOM_CFLAGS) -I. -Iexamples/vulkan $< -lm $(VULKAN_LIBS) -o $@
```